kliff/trainer/torch_trainer_utils/lightning_checkpoints.py

```diff
@@ -0,0 +1,92 @@
+import os
+
+import dill
+import pytorch_lightning as pl
+import torch
+
+from kliff.dataset import Dataset
+
+
+class SaveModelCallback(pl.Callback):
```

Comment:
Yes. Perhaps I can add it in next round of improvements. As it will include moving and deleting files on storage so need to be more cautious. Leaving this comment unresolved for a TODO later.